app/Resources/views/themes/tastefulplanet/form_builder/default_form_view.html.twig

```diff
@@ -0,0 +1,14 @@
+<section class="content-type-form full-view">
```

Comment:
Maybe `demo-form`? I believe you should consider using BEM naming convention consequently throughout all you code

Reply:
the name is consistent with the rest of the files (folder, product, place, article etc) on full and embeded view. Changing it to BEM will be great, we will create a separate PR for that, as global change on demo templates

Reply:
We will improve it as a separate issue https://jira.ez.no/browse/DEMO-237